templates/overrides/field/field--media--av-portal-video.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
```

Comment:
If we do this we can remove the workaround at oe_whitelabel_preprocess_paragraph__oe_av_media().

Reply:
True